<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connecter</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbDf1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>411</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnDisconnect">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Deconnecter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCommand">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEnvoyer">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblDf1Name">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListWidget" name="listReturn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>381</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="txtAddr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listTarget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>321</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
